Library/Formula/c.rb

```diff
@@ -0,0 +1,16 @@
+class C < Formula
+  homepage "https://github.com/ryanmjacobs/c"
+  url "https://github.com/ryanmjacobs/c/archive/v0.06.tar.gz"
+  sha1 "0eefb3523abccea7ebdd2c56d4d7be6f23585645"
+
+  depends_on "gnu-sed"
```

Comment:
Could you file an issue/patch about this? There doesn't seem to be any particular reason this should need `gnu-sed` over the one provided with OSX.